modules/jdbc/src/main/java/org/testcontainers/containers/JdbcDatabaseContainer.java

```diff
@@ -190,13 +215,13 @@ public void addParameter(String paramName, String value) {
      * @return startup time to allow, including image pull time, in seconds
      */
     protected int getStartupTimeoutSeconds() {
```

Comment:
`OracleContainer` is overriding this method, so this PR should also change `OracleContainer` to use fluent-setter to set it's default value.